src/views/partials/breaches.js

```diff
@@ -75,6 +75,9 @@ export const breaches = data => `
 <section>
   <header class='breaches-header'>
     <h1>${getMessage('breach-heading-email', { 'email-select': `<custom-select name='email-account'>${createEmailOptions(data.breachesData)}</custom-select>` })}</h1>
+    <button id='sendAlertTestButton' class='primary'>
+      Send test breach alert
+    </button>
```

Comment:
Only for testing the PR.

Reply:
The env `HIBP_NOTIFY_TOKEN` variable needs to be filled in [here](https://github.com/mozilla/blurts-server/pull/2790/files#diff-7b24ff39ac683ac7411b6527c5a510cfc036a9244fa613535460dbfcd734bc3dR115) for this to currently work. I’m just about to push a change that is handling this logic on the backend and should work out of the box.